recipes/qsum/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "qsum" %}
+  {% set version = "0.2.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/QCoding/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 'e5c01285550d9df96568b12e711d52074263d42f0ba4078dd3fd4aff69c201f6'
+
+build:
+  noarch: python
+  number: 0
+
+requirements:
+  build:
```

Comment:
For packages that don't use compilers, you don't need the `build` section, just `host` and `run`, so please remove `build` from here.